test/libsolidity/StandardCompiler.cpp

```diff
@@ -1505,6 +1505,47 @@ BOOST_AUTO_TEST_CASE(stopAfter_ast_output)
 	BOOST_CHECK(result["sources"]["a.sol"]["ast"].isObject());
 }
 
+BOOST_AUTO_TEST_CASE(dependency_tracking_of_abstract_contract)
```

Comment:
Can't this be a commandline test, i.e. see ``test/cmdlineTests/standard_yul_multiple_files`` for example.
But I may be missing a reason for not doing that...

Reply:
Well, hm - the IR output may be a bit fragile... I'm not sure...